app/models/form/lettings/subsections/property_information.rb

```diff
@@ -27,4 +27,8 @@ def pages
       Form::Lettings::Pages::PropertyMajorRepairsValueCheck.new(nil, nil, self),
     ].compact
   end
+
+  def displayed_in_tasklist?(log)
+    !log.supported_housing_and_renewal?
+  end
```

Comment:
This was the location of another error.
If we answer postcode_known and then alter the setup such that none of the questions in property information are routed to, tasklist.html.erb (l.12) tries to render the subsection as there is a question now considered applicable, however it then tries to make a link to the first question in that section that is routed to (none)
I have prevented that crash here by failing an earlier condition, but we may want to add safeguards elsewhere also, eg. in tasklist_helper:subsection_link